web_ui/src/pages/annotator/tools/edit-tool/edit-annotation-tool.component.tsx

```diff
@@ -32,21 +33,28 @@ const EditAnnotationToolFactory = ({
     disableTranslation = false,
     disablePoints = false,
 }: EditAnnotationToolFactoryProps) => {
-    const { roi } = useROI();
+    const { roi, image } = useROI();
     const { tasks, selectedTask } = useTask();
     const task = selectedTask ?? tasks[0];
+    const { scene } = annotationToolContext;
```

Comment:
Shouldn't we also use `useAnnotationToolContext` here?

Reply:
No need. If i did that then i would need to refactor way more branches than we need. This was the least footprint solution i could do. I did top down without touching almost no code. I added only 3 hooks on the labels component, which were using context from props.